csrc/python_frontend/fusion_cache.cpp

```diff
@@ -248,8 +249,8 @@ TrieNode* FusionCache::createChild(TrieNode* node, RecordFunctor* rec) {
           max_fusions_,
           "fusions.  The max_fusions for the FusionCache might need to be ",
           "increased if the max number is not being exceeded due to an error.");
-      fusions_.emplace_back(std::make_unique<FusionSchedules>());
-      fusion_id = fusions_.size() - 1;
+      fusion_id = fusions_.size();
```

Comment:
I'm guessing this is trying to be consistent with the default `fusion_id=0`?

Reply:
Yes, previously we required `fusion_id` to be greater than 0.